README.md

```diff
@@ -220,30 +274,23 @@ includes some additional security token, its duration should be limited to this
 expiration time to prevent callers from extending their access beyond the
 allowed time.
 
-The `req.satisfies(.., noReply)` method returns `true` if the client satisfies
-one of the scopesets. If the client does not satisfy one of the scopesets, it
-returns `false` and sends an error message unless `noReply` is true. 
-
-The first argument to `req.satisfies` can be a scopeset (in disjunctive normal
-form as described above).  Or, it can be an object, in which case the method
-will assume this object is a mapping from scope parameters (`<name>`) to
-values.
+The `async req.authorize(params, options)` method returns `true` if the client satisfies
```

Comment:
don't return `true`, I suggest we leave the return-value unspecified... In the future we might return `clientId` and set of scopes used to satisfy the expression... or something like that... as this is probably what taskcluster-auth will return...